css/layouts/_row.scss

```diff
@@ -0,0 +1,20 @@
+.l-row {
+  &.-equal-height {
+    margin-top: -#{map-get($grid-column-gutter, small)};
+
+    @media screen and (min-width: map-get($breakpoints, medium)) {
+      margin-top: -#{map-get($grid-column-gutter, medium)};
+    }
+
+    div[class*=col],
```

Comment:
Quick question, `div[class*=col]` will select all divs that can be selected by `div[class^=col]`, so why adding both selectors?